ios/Runner/AppDelegate.swift

```diff
@@ -9,6 +10,8 @@ import WebKit
         _ application: UIApplication,
         didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?
     ) -> Bool {
+        SwiftFlutterBackgroundServicePlugin.taskIdentifier = "your.custom.task.identifier"
```

Comment:
```suggestion
        SwiftFlutterBackgroundServicePlugin.taskIdentifier = "com.rtirl.chat.tts"
```
@kevmo314 Is this what I should change it to or should it be left as default?